File: C/temperatureConverter.c
#include <stdio.h>
 
int main()
{
  
    float fh,cl;
    int choice;
 
    printf("\n1: Convert temperature from Fahrenheit to Celsius."); 
    printf("\n2: Convert temperature from Celsius to Fahrenheit.");
    printf("\nEnter your choice (1, 2): ");
    scanf("%d",&choice);
 
    if(choice ==1){
        printf("\nEnter temperature in Fahrenheit: ");
        scanf("%f",&fh);
        cl= (fh - 32) / 1.8;
        printf("Temperature in Celsius: %.2f",cl);
    }
    else if(choice==2){
        printf("\nEnter temperature in Celsius: ");
        scanf("%f",&cl);
        fh= (cl*1.8)+32;
        printf("Temperature in Fahrenheit: %.2f",fh);
    }
    else{
        printf("\nInvalid Choice !!!");
    }
    return 0;
}

File: Python/Temperature_converter.py
# Created by BIGYAN RIJAL (Github: @theoptimist76)

# Enter the value to convert in proper convention
temp = input("Input the  temperature you would like to convert? (e.g. 45F, 102C etc.) : ")
degree = int(temp[:-1])
input = temp[-1]

#Checking if the value of temperature is in celcius or farhenheit 
if input.upper() == "C":
  result = int(round((9 * degree) / 5 + 32))
  output = "Fahrenheit"
elif input.upper() == "F":
  result = int(round((degree - 32) * 5 / 9))
  output = "Celsius"

# Checking proper convention of temperature values
else:
  print("Input proper convention of temperature.")
  quit()
  
# Display the output in celcius or farhenheit and vice versa
print("The temperature in", output, "is", result, "degrees.")
